### PR TITLE
Don't let your actions harm Earth

### PR DESCRIPTION
**Pull request by yashikacac**

### CODE REVIEW

File: _hackbars/yashikacac.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+layout: hackbar
+title: Open Source
+author: yashikacac
+---
+
+Help climate grow this hacktober fest using your developer skills!
+
+---
+
+## <!-- Your MESSAGE on climate change -->
```
Comment (tasnimzotder) [inline _hackbars/yashikacac.md]: hey @yashikacac, Add your message on `climate change` and make a commit

Comment (yashikacac) [reply _hackbars/yashikacac.md]: sure

Comment (tasnimzotder) [reply _hackbars/yashikacac.md]: Thanks

Comment (yashikacac) [reply _hackbars/yashikacac.md]: May you please let me know whether is it done or not.

Comment (tasnimzotder) [reply _hackbars/yashikacac.md]: it's done!

**Review by tasnimzotder — APPROVED** [review]
Thanks @yashikacac for your contribution :)

**Pull request merged**